src/v2/layouts/HomeScreen/HomeScreen.tsx

```diff
@@ -32,10 +31,7 @@ export const HomeScreen = (): JSX.Element => {
 						</StyledImage>
 					</StyledLogoWrapper>
 					<StyledButtonContainer>
-						{/*TODO:- Need to handle navigation here.*/}
-						<StyledButton onClick={() => console.log('data')}>
-							New Registration
-						</StyledButton>
+						<LinkStyled to="/member-register">New Registration</LinkStyled>
```

Comment:
User exported enum for Routes here. 
```suggestion
						<LinkStyled to={PageRoutes.MEMBER_REGISTRATION }>New Registration</LinkStyled>
```